docs/guide/deployment.md

```diff
@@ -261,7 +261,35 @@ npm install -g now
 
 ### Heroku
 
-> TODO | Open to contribution.
+1. [install HerokU CLI](https://devcenter.heroku.com/articles/heroku-cli)
+
+2. Create a `static.json` file:
+```json
+{
+  "root": "dist",
+  "clean_urls": true,
+  "routes": {
+    "/**": "index.html"
+  }
+}
+```
+
+3. Add `static.json` file to git
+```bash
+git add static.json
+git commit -m "add static configuration"
+```
+
+4. Deploy to Heroku
+```bash
+heroku login
+heroku create
+heroku buildpacks:add heroku/nodejs
+heroku buildpacks:add https://github.com/hone/heroku-buildpack-static
```

Comment:
```suggestion
heroku buildpacks:add https://github.com/heroku/heroku-buildpack-static
```
Previous URL redirects to Heroku organisation repo now.

Reply:
Nice, thanks!